examples/cosmos-db/main.tf

```diff
@@ -18,10 +18,13 @@ resource "azurerm_cosmosdb_account" "db" {
 
     enable_automatic_failover = true
 
+    //set ip_range_filter to allow azure services (0.0.0.0) and azure portal.
+    ip_range_filter = "0.0.0.0,104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26"
```

Comment:
AFAIK this should only need to be `0.0.0.0`, as the Azure Portal IP Addresses will change depending on where you're located?

Reply:
These are the IPs I got when I selected the option in the azure portal.